kustomize/test_kustomizations/_example_app/ingress.yaml

```diff
@@ -9,6 +9,7 @@ spec:
   - http:
       paths:
       - path: /testpath
+        pathType: ImplementationSpecific
```

Comment:
TODO: Determine if this field is required, or if this is an edge case. If `pathType` is not set, the provider detects it as a change on every plan.

Reply:
This field is definitely required on Ingress v1

Reply:
This might not be a particularly unusual edge case - kubernetes adds fields to resource specs that then appear to be missing in future runs. This isn't great here, but is likely to be much worse for other resource types (I think maybe Service in particular)